poster/poster.tex

```diff
@@ -0,0 +1,428 @@
+			NFCSim \cite{schneider_nfcsim:_2005}& LANL &  D-D & PRED \\
+			NGSAM \cite{aubin_development_2013} & ORNL & NONE & M \\
+			{\small NUWASTE \cite{garrick_nuclear_2011}}& NWTRB &  M & M \\
+			ORION \cite{feng_standardized_2016} & NNL & CONST  & PROP\\
+			VISION \cite{feng_standardized_2016,boucher_international_2010}& INL  & D-D & PROP \\
+			VISTA \cite{iaea_nuclear_2007}& IAEA & D-D & PROP \\ \hline
+		\end{tabularx}
+		\caption{Simulators, categorized by their reactor and fuel cycle 
+			facility deployment strategies.}
+		\label{tam:sim-deploy}
+	\end{table}
+	
+\end{block}
+
+%----------------------------------------------------------------------------------------
+
+\end{column} % End of column 2.1
+
+\begin{column}{\onecolwid} % The second column within column 2 (column 2.2)
+
+%----------------------------------------------------------------------------------------
+%	RESULTS
+%----------------------------------------------------------------------------------------
+
+\begin{block}{Promising Algorithms}
+ \begin{itemize}
+ 	\setlength\itemsep{1.5em}
+	\item \textbf{{\Large Non-Optimizing (NO)}}
+	\begin{itemize}
+		\setlength\itemsep{1em}
+		\item {\large Predict based on historical supply-demand data}
+		\item{\large  Does not attempt to meet demand optimally}
+		\item {\large Fast execution time with limited precision}
+		\item {\large ex) Autoregressive Moving Average (ARMA), Autoregressive Conditional Heteroskedastic (ARCH)}
+	\end{itemize}
+	\item \textbf{{\Large Deterministic Optimizing (DO)}}
+	\begin{itemize}
+		\setlength\itemsep{1em}
+		\item{\large  Minimize or Maximize an objective function with set of constraints}
+		\item {\large Replicable}
+		\item{\large  ex) Global Change Assessment Model (GCAM), MARKet and ALlocation (MARKAL)}
+	\end{itemize}
+	\item \textbf{{\Large Stochastic Optimizing (SO)}}
+	\begin{itemize}
+		\setlength\itemsep{1em}
+		\item {\large Probabilistic search into the objective function or constraint models}
+		\item {\large Uncertainty in addition to mean}
+		\item {\large Uses random samples from probability distributions}
+		\item {\large ex) Markov Switching-Model, Gaussian Process Regression}
+	\end{itemize}
+\end{itemize}
+
+
+\end{block}
+
+
+
+%----------------------------------------------------------------------------------------
+
+\end{column} % End of column 2.2
+
+\end{columns} % End of the split of column 2
+
+\end{column} % End of the second column
+
+\begin{column}{\sepwid}\end{column} % Empty spacer column
+
+\begin{column}{\onecolwid} % The third column
+
+%----------------------------------------------------------------------------------------
+%	CONCLUSION
+%----------------------------------------------------------------------------------------
+
+\begin{block}{Conclusion}
+The review concludes that fuel cycle simulation tools approach scenario objective 
+functions in various ways.
+\begin{itemize}
+	\item wrap in external optimizer
```

Comment:
this is an commanding imperative phrase, while the following bulled is a noun phrase. Bullets should have the same grammatical form and unless they are independent sentences, they should be part of a sentence (instead of a period after ways, have a colon.)

Reply:
would `wrap->wrapping` make sense?

Reply:
It's probably better to change the second bullet to become a verb (`predict deployment strategy with look-ahead methods`)